docs/appendices/release-notes/unreleased.rst

```diff
@@ -52,6 +52,30 @@ Breaking Changes
   Docker, or via a service manager like ``systemd`` where these options are not
   required.
 
+- Changed subtraction of timestamps to not return their difference in
+  milliseconds but return an ``INTERVAL`` value instead.
```

Comment:
Part of the the problem was that it was typed as `timestamp`, so in clients using the type information you got a garbage result. Maybe worth highlighting. 

Changing it to `long` as diff in milliseconds would also have been an option

Reply:
But from my test, it was typed as `bigint` not `timestamp`.

Reply:
Are you sure?

The function registration used the same type for argument and return value:

```
module.register(
                    Signature.scalar(
                        op.toString(),
                        type.getTypeSignature(),
                        type.getTypeSignature(),
                        type.getTypeSignature()
                    ).withFeatures(op.features),
                    (signature, boundSignature) ->
                        new BinaryScalar<>(op.longFunction, signature, boundSignature, type)
```

```
cr> select pg_typeof( ('2022-01-03'::timestamp - '2022-01-01'::timestamp));
+-------------------------------+
| 'timestamp without time zone' |
+-------------------------------+
| timestamp without time zone   |
+-------------------------------+
```

Reply:
You're right the output is misleading:
```
{15:33}/tmp/crate-5.1.1 ➭ crash --host "localhost:4200" --verbose
+-----------------------+---------------+---------+-----------+---------+
| server_url            | node_name     | version | connected | message |
+-----------------------+---------------+---------+-----------+---------+
| http://localhost:4200 | Monte Cernera | 5.1.1   | TRUE      | OK      |
+-----------------------+---------------+---------+-----------+---------+
CONNECT OK
CLUSTER CHECK OK
TYPES OF NODE CHECK OK
cr> SELECT '2022-12-05T11:22:33.123456789'::timestamp - '2022-11-21T10:11:22.0012334'::timestamp;                                                                                    
+--------------------+
| 1213871122::bigint |
+--------------------+
|         1213871122 |
+--------------------+
SELECT 1 row in set (0.006 sec)
cr>                                                                                                                                                                                  
Bye!
{15:33}/tmp/crate-5.1.1 ➭ psql -h localhost -p 5433 -U crate       
psql (14.6, server 10.5)
Type "help" for help.

crate=> SELECT '2022-12-05T11:22:33.123456789'::timestamp - '2022-11-21T10:11:22.0012334'::timestamp;    
     1213871122::bigint     
----------------------------
 1970-01-15 01:11:11.122+00
(1 row)

crate=> SELECT pg_typeof('2022-12-05T11:22:33.123456789'::timestamp - '2022-11-21T10:11:22.0012334'::timestamp);    
 'timestamp without time zone' 
-------------------------------
 timestamp without time zone
(1 row)
```